package/requirements.txt

```
@@ -6,3 +6,5 @@ fastapi>=0.63.0, <0.67.0
 aiofiles==0.6.0
 uvicorn~=0.13.4
 watchgod~=0.7
+plotly>=4.14.3
```

We could make this a bit less strict and do:
```suggestion
plotly~=4.14
```
that way it would be pinned the same way as in Kedro core.